apps/dialtone-documentation/docs/components/dropdown.md

```diff
@@ -9,9 +9,74 @@ figma_url: https://www.figma.com/file/2adf7JhZOncRyjYiy2joil/DT-Core%3A-Componen
 ---
 
 <code-well-header bgclass="d-bgc-neutral-white">
```

Comment:
Don't think we want to use neutral white here anymore.

![Screenshot 2024-03-26 at 2 52 53 PM](https://github.com/dialpad/dialtone/assets/64808812/e551c593-a325-4633-ba2d-2428bbfd8611)

Reply:
Definitely need a better example. Plus, there's no hover or active behavior. Use this one to demo the variety of a Dropdown. 
<img width="223" alt="image" src="https://github.com/dialpad/dialtone/assets/1165933/b6126351-3ba1-4571-9858-cd854f2f95eb">

Later I'd like some better examples of Dropdowns, e.g with icons and such (and of course better Design-specific documentation, but I digress).

Reply:
The styles aren't appearing, e.g. `.dt-list-separator` and style interactions for `:focus`, `:hover`, `:active`. 

https://github.com/dialpad/dialtone/assets/1165933/83648252-06a9-4b59-9dad-cb1f0976bb39